MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,298 @@
+import Foundation
+import UIKit
+import PureLayout
+import MovieAppData
+
+class MovieDetailsViewController: UIViewController {
+    
+    private var movieImageView: UIImageView!
+    
+    private var userScoreLabel: UILabel!
+    private var movieTitleLabel: UILabel!
+    private var releaseDateLabel: UILabel!
+    private var genreLabel: UILabel!
+    private var movieLengthLabel: UILabel!
+    private var favoriteButton: UIButton!
+    private var overviewLabel: UILabel!
+    private var summaryLabel: UILabel!
+    private var crewView: UICollectionView!
+    
+    
+    private var screenHeight: CGFloat!
+    private var screenWidth: CGFloat!
+    private var movieDetails: MovieDetailsModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        movieDetails = MovieUseCase().getDetails(id: 111161)
+        
+        createViews()
+        customizeViews()
+        defineViewLayout()
+        
+    }
+    
+    
+    private func createViews() {
+        
+        screenHeight = view.frame.height
+        screenWidth = view.frame.width
+        
+        
+        movieImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: screenWidth, height: screenHeight*0.4))
+        userScoreLabel = UILabel(frame: CGRect(x: screenWidth*0.06, y: screenHeight*0.16, width: screenWidth, height: 20))
```

Comment:
Why setup like this? Why do you use frames? Can you do this by setting up constraints?